packages/@react-spectrum/menu/intl/ar-AE.json

```
@@ -1,4 +1,5 @@
 {
   "moreActions": "المزيد من الإجراءات",
-  "unavailable": "غير مُتوفر، قُم بالتوسيع للحصول على التفاصيل"
+  "unavailable": "غير مُتوفر، قُم بالتوسيع للحصول على التفاصيل",
+  "backButton": "Return to "
```

SubMenu tray back button has the above as its aria-label and then is aria-labelledby a neighboring span that contains the text of its submenu trigger. See build attached to [66ce969](https://github.com/adobe/react-spectrum/pull/4976/commits/66ce969411c91d333a86eb901ac1f69eabda7f3d) (will add link when build finishes

---

Is there a way we can revert back to this? (without getting double announcements)

```suggestion
  "backButton": "Return to {prevMenuButton}"
```

Not sure the relying on the aria-labelledby will make sense in some locales in terms of ordering.

---

I don't quite remember if the ordering here is actually a problem in other locales, do we have an example? I noticed a bunch of other components use the same labeling in LTR as RTL (aka useColorArea, etc)

---

One example is in dnd:

en-US: `"dragItem": "Drag {itemText}"`
de-DE: `"dragItem": "{itemText} ziehen"`

and

en-US: `"dropOnItem": "Drop on {itemText}"`
de-DE: `"dropOnItem": "Auf {itemText} ablegen"`.

---

Sweet thanks, I can def revert to my old approach then